frontend/src/views/pages/vehicle/Vehicle.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <!-- <CButton color="secondary" @click="$router.go(-1)" class="mb-2"
+      >Go back to the inventory</CButton
+    > -->
```

Comment:
Remove commented code

Reply:
Will create a pull request next sprint that will clean up all the console.logs and error cases for API requests